pkg/networkservice/common/excludedprefixes/server_test.go

```diff
@@ -59,7 +59,7 @@ func TestNewExcludedPrefixesService(t *testing.T) {
 }
 
 func TestCheckReloadedPrefixes(t *testing.T) {
-	prefixes := []string{"172.16.1.0/24", "10.32.0.0/12", "10.96.0.0/12"}
+	prefixes := []string{"10.32.0.0/12", "10.96.0.0/12", "172.16.1.0/24"}
```

Comment:
Why tests have changed? Looks like it is more correct, but could you please clarify this :)

Reply:
IPPool sort subnets with the bit order within it's RB tree, so I changed prefixes order to check in the test